Source/WebCore/rendering/style/GridArea.h

```diff
@@ -200,6 +199,8 @@ class GridSpan {
         m_endLine = std::max(GridPosition::min() + 1, std::min(endLine, GridPosition::max()));
     }
 
+private:
```

Comment:
I needed to create a span, so I exposed the constructor. Maybe there is a better way of doing this.

Reply:
I'm not 100% sure if this is ok (would need to think about it more) but for your usecase can you just use the static `translatedDefiniteGridSpan` function instead?

Reply:
Yeah, we can probably just use that instead 😀